spec/spec_helper_acceptance.rb

```diff
@@ -22,14 +22,15 @@
   c.before :suite do
     # Install module and dependencies
     hosts.each do |host|
-      if fact_on(host, 'osfamily') == 'RedHat'
+      if fact_on(host, 'os.family') == 'RedHat'
         # don't delete downloaded rpm for use with BEAKER_provision=no +
         # BEAKER_destroy=no
         on host, 'sed -i "s/keepcache=.*/keepcache=1/" /etc/yum.conf'
         # refresh check if cache needs refresh on next yum command
         on host, 'yum clean expire-cache'
+        major = fact_on(host, 'os.release.major')
         # We need SCL as well for dynflow
-        if fact_on(host, 'operatingsystem') == 'CentOS'
+        if fact_on(host, 'operatingsystem') == 'CentOS' && major == '8'
```

Comment:
`== 7` is what we want here :)

Reply:
I'll take a look at incorporating that in https://github.com/theforeman/puppet-foreman_proxy/pull/581